.jenkinsci/choose-platform.groovy

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env groovy
+
+// choose platform to count coverage
+def choosePlatform() {
+  if (params.Merge_PR) {
+    return CoveragePlatforms.x86_64_aws_cov.toString()
+  }
+  if (params.Linux) {
+    return CoveragePlatforms.x86_64_aws_cov.toString()
+  }
+  if (!params.Linux && params.MacOS) {
+    return CoveragePlatforms.mac.toString()
```

Comment:
The requirement is to collect coverage only on gcc/Linux, so maybe mac, armv8, and armv7 can be removed?

Reply:
@lebdron according to the requirements, this file is not needed anymore. It's removed in the next commit